### PR TITLE
PLATFORM-596: Adding Account Generator to the pipeline

### PR DESCRIPTION
**Pull request by mkrielza**
We would need the same pipeline integration for the account generator (even when it is just a tool).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+          key: deps-{{ checksum "build.gradle" }}-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - .gradle
+            - ~/.gradle
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - ./
+
+#  acceptanceTests:
```
Comment (fllaca) [inline .circleci/config.yml]: Remove or add a TODO comment for this commented code

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+      - publish_docker:
+          requires:
+            - build_docker
+          filters:
+            branches:
+              only:
+                - master
+                - /^release-.*/
+
+# TODO remove
```
Comment (fllaca) [inline .circleci/config.yml]: can we  remove this?

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ENTRYPOINT ["/opt/accountgenerator/bin/accountgenerator"]
+
+# Build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.name="accountgenerator" \
+      org.label-schema.description="Ethereum account generator application" \
+      org.label-schema.url="https://docs.accountgenerator.pegasys.tech/" \
```
Comment (fllaca) [inline docker/Dockerfile:20]: is this `pegasys.tech` ? should it be an adhara url instead?

File: docker/cloudhsm.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ENTRYPOINT ["/opt/accountgenerator/bin/cloudhsm-entrypoint.sh"]
+
+# Build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.name="accountgenerator" \
+      org.label-schema.description="Ethereum account generator application" \
+      org.label-schema.url="https://docs.accountgenerator.pegasys.tech/" \
```
Comment (fllaca) [inline docker/cloudhsm.Dockerfile:33]: same as above? change to an adhara url?

**Review by fllaca — COMMENTED** [review]
lgtm, just very minor aesthetics

File: docker/Dockerfile (hunk 3 above)
Comment (mkrielza) [reply docker/Dockerfile:20]: The tool was created solely to generate accounts to be used with Pegasys EthSigner. Most of its non-crypto code was taken from EthSigner to be able to function exactly like the EthSigner JSON RPC server. I can remove all reference to Pegasys if that is best?

**Pull request merged**